yarn.lock

```diff
@@ -6246,7 +6246,7 @@ ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.2, ajv@^6.12.3, ajv@^6.12.4, ajv
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
-ajv@^8.6.3:
+ajv@^8.12.0, ajv@^8.6.3:
```

Comment:
I think you mentioned concern about this in standup but this should mean that packages specify ^8.12.0 and ^8.6.3 but the resolved version is 8.12.0. Both specify they are accepting any minor version bumps [with the ^ syntax](https://docs.npmjs.com/about-semantic-versioning). The resolved version ended up being 8.12. So there isn't a concern of multiple versions being pulled in as both are flexible with minor versions.

Reply:
The package above this one is also ajv:
`ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.2, ajv@^6.12.3, ajv@^6.12.4, ajv@^6.12.5:`

Reply:
I was specifically looking at your changes and how your change does not pull in an additional version. A quick glance at the yarn lock file, it looks like we were already using multiple versions. Your changes have no impact on that. You specified you need 8.12.0, we were already using 8.12.0 since the request for 8.6.3 resolved to 8.12.0 since it allows minor version bumps. 

Here's the information yarn reports if you're curious why we have more than one but your changes are not adding any new packages according to that `yarn.lock` file change, it's just resolving your dependency to one that already exists, specifically the one that the swagger tooling we're already using pulled in:

```
yarn why ajv
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "ajv"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "ajv@6.12.6"
info Has been hoisted to "ajv"
info Reasons this module exists
   - "workspace-aggregator-dee04f1f-be9a-4c89-8c02-a6e3daf0a502" depends on it
   - Hoisted from "_project_#eslint#ajv"
   - Hoisted from "_project_#webpack#ajv"
   - Hoisted from "_project_#schema-utils#ajv"
   - Hoisted from "_project_#webpack#schema-utils#ajv"
   - Hoisted from "_project_#fork-ts-checker-webpack-plugin#schema-utils#ajv"
   - Hoisted from "_project_#eslint#@eslint#eslintrc#ajv"
   - Hoisted from "_project_#css-loader#schema-utils#ajv"
   - Hoisted from "_project_#style-loader#schema-utils#ajv"
   - Hoisted from "_project_#back-end#clickhouse#request#har-validator#ajv"
   - Hoisted from "_project_#front-end#@storybook#builder-webpack5#babel-loader#schema-utils#ajv"
info Disk size without dependencies: "1.12MB"
info Disk size with unique dependencies: "1.89MB"
info Disk size with transitive dependencies: "1.93MB"
info Number of shared dependencies: 5
=> Found "@apidevtools/swagger-parser#ajv@8.12.0"
info This module exists because "_project_#back-end#@apidevtools#swagger-parser" depends on it.
info Disk size without dependencies: "2.48MB"
info Disk size with unique dependencies: "3.18MB"
info Disk size with transitive dependencies: "3.22MB"
info Number of shared dependencies: 5
✨  Done in 1.18s.
```

The back-end is using version 6 for the clickhouse validation looks like, and you're making an existing dev dependency of 8 a production dependency for the front-end. If you need to add this functionality to the back-end then we would have 2 versions on the back-end running in prod, if that happened, you'd continue to use whichever one you specified, and the clickhouse request validator would use the one it needs.

I don't think you need to worry about this in terms of version conflicts, and we can't avoid adding to the front-end bundle if you need it in prod. Maybe I'm not clear on what your concerns are but let me know if you had a specific question/concern I can help with.